test/snapshot-order/test.js

```diff
@@ -0,0 +1,52 @@
+const test = require('@ava/test');
+const exec = require('../helpers/exec');
+const fs = require('fs');
+const path = require('path');
+
+test('snapshot files are independent of test resolution order', async t => {
+	const options = {
+		cwd: exec.cwd('intertest-order')
+	};
+	// Run, updating snapshots.
+	await exec.fixture(['test.js', '-u'], options);
```

Comment:
```suggestion
	await exec.fixture(['test.js', '--update-snapshots'], options);
```

Same in other places.